src/main/java/uk/gov/ons/ssdc/caseprocessor/model/dto/EnrichedSmsFulfilment.java

```diff
@@ -9,4 +10,5 @@
   private String packCode;
   private String uac;
   private String qid;
+  private Map<String, String> uacMetadata;
```

Comment:
Should be `Object` or `String` to allow future flexibility beyond flat key:value pairs of strings